internal/graphql/schema/schema.graphqls

```diff
@@ -18,6 +18,7 @@ enum MeshType {
 enum Status {
 	ENABLED
 	DISABLED
+	PROCESSING
```

Comment:
Will these states be used in the Messaging System; used universally across all operations performed by Meshery Adapters?

Why are these not centrally defined?

Reply:
We cannot separate and import schemas from different sources. A graphql limitation of the relay plugin.